website/src/_data/capability-matrix.yml

```diff
@@ -18,8 +18,10 @@ columns:
     name: Google Cloud Dataflow
   - class: flink
     name: Apache Flink
-  - class: spark
-    name: Apache Spark
+  - class: spark-rdd
+    name: Apache Spark (RDD/DStream based)
+  - class: spark-ss
+    name: Apache Spark (Structured Streaming based)
```

Comment:
What do you think about name it as `Apache Spark (Dataset based)`?

Reply:
+1

Reply:
In this case, I'd leave it as it was before - `name: Apache Spark (Structured Streaming based)`. Otherwise, it's too long and looks very piled in the table column. 
My proposal was to rename it to `Apache Spark (Dataset based)` and `spark-dataset` as short term everywhere instead of 'spark-ss'. I belirve that `spark-dataset` already says my its name, `spark-ss` is not clear what does it mean at the first sight.